string name
int32 index
int32 action
float64 likelihood
